release.py

```diff
@@ -348,6 +348,9 @@ def command_release_announcement_email():
 
 <DETAILS OF THE CHANGES>
 
+An overview of the release and its notable changes can be found in the
+release blog post: <LINK TO THE KAFKA BLOG>
```

Comment:
Can we generate the link automatically if the link formate is like `https://kafka.apache.org/blog#apache_kafka_362_release_announcement`

Reply:
Thanks for the review,. updated the PR